project/frontend/src/App.js

```diff
@@ -123,95 +128,14 @@ function App() {
       ></Route>
       <Route
         path="/competition"
-        render={() => (
-          <Competition
-          ></Competition>
-        )}
+        render={() => <Competition></Competition>}
```

Comment:
If we're not passing data to elements, may as well just write it as 
```js
<Competition />
```

There's a few other instances where this could be done too

Reply:
Done.